src/components/Navbar.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { MdClose } from 'react-icons/md';
+import { FiMenu } from 'react-icons/fi';
+
+const Navbar = () => {
+  const [navbarOpen, setNavbarOpen] = useState(false);
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Home',
+    },
+    {
+      id: 2,
+      path: '/about',
+      text: 'About',
+    },
+  ];
+
+  const handleToggle = () => {
+    setNavbarOpen(!navbarOpen);
+  };
+
+  const closeMenu = () => {
+    setNavbarOpen(false);
+  };
+
+  return (
+    <nav className="navBar">
+      <button type="button" onClick={handleToggle}>
+        {navbarOpen ? (
+          <MdClose style={{ color: '#fff', width: '40px', height: '40px' }} />
+        ) : (
+          <FiMenu style={{ color: '#7b7b7b', width: '40px', height: '40px' }} />
+        )}
```

Comment:
![review](https://user-images.githubusercontent.com/92215878/171105657-68ff5366-397e-455c-8cf9-50948a06275d.png)

- [ ] The hamburger icon is overlapping with the nav items. Please make sure that it's working as intended or remove the icon.